site/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: site
+resources:
+- namespace.yaml
```

Comment:
inconsistent indentation style with the rest of the PR

Reply:
Is there an autoformatter for these?

Reply:
should be able to find any old yaml formatter...